ocs_ci/helpers/helpers.py

```diff
@@ -170,7 +170,10 @@ def create_pod(
         AssertionError: In case of any failure
 
     """
-    if interface_type == constants.CEPHBLOCKPOOL:
+    if (
+        interface_type == constants.CEPHBLOCKPOOL
```

Comment:
dont we need to use different pools for RBD and RBD-Thick ?

Reply:
@Avilir  - In this function  we are creating a pod. 

As for your question - when the test runs for RBD interface and RBD - thick interfaces those are separate runs and we will not have case in this the same pool contains pvcs of different types. 
What differs is storage class, not the pool.